chart/open-feature-operator/README.md

```diff
@@ -82,58 +82,59 @@ helm uninstall open-feature-operator
-| `managerConfig.controllerManagerConfigYaml`                 | `1`                                          |                                                                                                                                                                                                                                  |
-| `managerConfig.replicas.health.healthProbeBindAddress`      | `:8081`                                      |                                                                                                                                                                                                                                  |
-| `managerConfig.replicas.metrics.bindAddress`                | `127.0.0.1:8080`                             |                                                                                                                                                                                                                                  |
-| `managerConfig.replicas.webhook.port`                       | `9443`                                       |                                                                                                                                                                                                                                  |
-
-### flagd-proxy Configuration
-
-| Value                                              | Default                                      | Explanation                                                                       |
-| -------------------------------------------------- | -------------------------------------------- |---------------------------------------------------------------------------------- |
-| `flagdProxyConfiguration.port`                 | `8015`                                       | Sets the port to expose the sync API on                                           |
-| `flagdProxyConfiguration.metricsPort`          | `8016`                                       | Sets the port to expose the metrics API on                                        |
-| `flagdProxyConfiguration.image.repository`     | `ghcr.io/open-feature/flagd-proxy`      | Sets the image for the flagd-proxy deployment                                |
-| `flagdProxyConfiguration.image.tag`            | current flagd-proxy version: `v0.2.0`   | Sets the tag for the flagd-proxy deployment                                  |
-| `flagdProxyConfiguration.debugLogging`         | `false`                                      | Controls the addition of the `--debug` flag to the container startup arguments    |
-
-## Changelog
-
-See [CHANGELOG.md](https://github.com/open-feature/open-feature-operator/blob/main/CHANGELOG.md)
```

Comment:
the Changelog section is lost with these changes. I suggest moving it before the Parameter section.

Reply:
Personally I do not think that this is a good place for changelog link, maybe to root readme will be better ?

Reply:
I moved it to the main README, hope that will be ok